client/src/components/LayoutComponent/Form/index.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { Fragment, Component } from 'react';
+import {
+	Checkbox,
+	FormControlLabel,
+	Grid,
+	TextField,
+	Select,
+	MenuItem,
+	InputLabel,
+	Button,
+	FormControl,
+} from '@material-ui/core';
+import axios from 'axios';
+import Alert from '@material-ui/lab/Alert';
+import Dialog from '../../CommonComponent/Dialog';
+
+class Form extends Component {
+	state = {
+		questions: [],
+		newQuestion: '',
+		type: '',
+		loading: true,
+		Error: '',
+		typeError: '',
+		open: false,
+	};
+
+	componentDidMount = () => {
+		axios
+			.get('/dashboard/form')
+			.then(({ data }) => {
+				if (!data) return this.setState({ Error: 'There is no data' });
+				this.setState({ questions: data, loading: false });
+			})
+			.catch((err) => console.log(err));
+	};
+
+	handleChange = (index, event) => {
+		let { name, value, type } = event.target;
+		if (type === 'checkbox') {
+			const newCheckedArry = this.state.questions.map((question) => {
+				if (index === question.id) {
+					question.checked = !question.checked;
+				}
+				return question;
+			});
+
+			return this.setState({ questions: newCheckedArry });
+		}
+		this.setState({ [name]: value });
+	};
+	handleSelect = (event) => {
+		this.setState({ type: event.target.value });
+	};
+	handleClick = (event) => {
```

Comment:
`handleAddQuestionClick`

otherwise i don't know what the button is meant to do!